proxies/live/apiproxy/policies/AssignMessage.SetAuthorizationHeader.xml

```diff
@@ -0,0 +1,9 @@
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.SetAuthorizationHeader">
+  <Set>
+    <Headers>
+      <Header name="Authorization">{accesstoken.external_access_token}</Header>
```

Comment:
I think you need to add `Bearer ` to the start of the header value for this to work

Reply:
Of course I do.

I had this right the first time I wrote it, but I've rewritten it twice since then and broken it all :joy: